intelmq/bots/experts/url2fqdn/expert.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import sys
+from urlparse import urlparse
```

Comment:
Use these lines for urlparse import: https://github.com/certtools/intelmq/blob/master/intelmq/lib/harmonization.py#L34
This will work in all versions.